algorithmic_efficiency/workloads/mnist/mnist_pytorch/workload.py

```diff
@@ -73,8 +73,13 @@ def _build_input_queue(
             batch['inputs'], dtype=torch.float32, device=DEVICE)
         targets = torch.as_tensor(
             batch['targets'], dtype=torch.long, device=DEVICE)
-        weights = torch.as_tensor(
-            batch['weights'], dtype=torch.bool, device=DEVICE)
+        if 'weights' in batch:
+          weights = torch.as_tensor(
+              batch['weights'], dtype=torch.bool, device=DEVICE)
+        else:
+          weights = torch.ones((batch['targets'].shape[-1],),
```

Comment:
Superfluous weights will also be broadcasted during training, but I think it's ok since the overhead is probably negligible.

Reply:
Yes, I think this should be fine for MNIST, as the timings for PyTorch are roughly the same as Jax.